dev-python/confuse/confuse-1.0.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
```

Comment:
There's no python3.4 in Gentoo tree anymore. Wonder why the CI bot didn't yell at you for this...?

Reply:
Done!